api/email/emailModel.js

```diff
@@ -113,4 +114,27 @@ async function saveResetCode(uid, code)
     {
         console.log(ex); return -1;
     }
+}
+
+// Regex checking if the username only contains letters and numbers
+async function userNamePattern(username) 
+{
+    {
+        const codenamePattern = /^[A-Za-z0-9]*$/;
+        try 
+        {
+            if (codenamePattern.test(username)) 
+            {
+                return true
+            }
+            else 
+            {
+                return false
+            }
+        }
+        catch (error) 
+        {
+            console.log(error)   
+        }
+    }
```

Comment:
Is there an extra bracket encasing here?

Reply:
@lgv-0 if a try catch is not needed then we should be good once your type validation aligns with my vector